cs/forms.texy

```diff
@@ -335,9 +349,8 @@ $form->addButton('raise', 'Zvýšit plat')
 \--
 
 
-
-addImage($name, $alt = NULL) .{toc: addImage()}
------------------------------------------------
+addImageButton($name, $alt = NULL) .{toc: addImage()}
+-----------------------------------------------------
```

Comment:
Damn, thanks. It was in the EN version, found it [here](http://api.nette.org/2.0.18/Nette.Forms.Container.html) aswell.